packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -95,6 +95,17 @@ export class PipelineService {
     return metadataArr.find(r => r['name'] === name)['schema_name'];
   }
 
+  /**
+   * Creates a Dialog for passing to makeServerRequest
+   */
+  static getWaitDialog(): Dialog<any> {
+    return new Dialog({
```

Comment:
Since this dialog is created by different tasks, could you make title and body text optionally customizable? As is, the same dialog would be displayed.